### ISSUE TITLE
Is this still supporting?

### ISSUE THREAD

**Issue by willian** [upmin/upmin-admin-ruby#170]
Hi!
Based on the number of open issues and pull requests, I'm wondering if you are still maintaining this project.

Thanks,

**Comment by jdurand** [comment]
@willian I decided to use UpminAdmin for a long term project of mine. @joncalhoun gave me baby sitting rights and I'll do my best to support and contribute as my project now depends on it.
I'll dig more into this in the upcoming weeks.

**Comment by willian** [comment]
Thanks for the answer @jdurand.
I need to install an Admin for my client. I have used ActiveAdmin a lot, then I found UpminAdmin. But I don't have time now to learn it or to understand all the main differences between the both systems.

Could you tell me what does UpminAdmin does or doesn't comparing to ActiveAdmin?

Best,

**Comment by jdurand** [comment]
UpminAdmin is lighter.

I've used ActiveAdmin a lot. The main reasons why I decided to invest time in UpminAdmin : 
* AA is a DSL (based on Arbre) where UA is model decorators and drop-in view partials (haml, erb, whatever really)
* AA gives you more out of the box, but UA customization is more flexible
* If you really want to customize AA you'll either push it's DSL to the limits (and see your forms disappear for some reason) or resort to use custom templates which I think defeats the purpose of using AA in the first place.

One gotcha: UA does not react well to having models prefixed with _admin_ (i.e. admin_user)

Try it out and make a list of pain points. I have mine. When I get my hands dirtier, I'll try go over everything.
